packages/common/package.json

```diff
@@ -25,6 +25,7 @@
     "clean:build": "bux rmrf ./build",
     "clean:esm": "bux rmrf ./lib",
     "clean:cjs": "bux rmrf ./lib-cjs",
+    "clean:localization": "bux rmrf ./src/ui-common/localization/generated",
```

Comment:
If we do switch this so `generated` is at the top, this should probably be `clean:generated`

Reply:
I removed the `clean:localization` command altogether (as cleaning out the localization files via `workspace:clean` erases the translations after they are built, same issue that this PR fixed: https://github.com/Azure/BatchExplorer/pull/2775)